Makefile

```diff
@@ -1,33 +1,41 @@
+# variables that should not be overridden by the user
 VERSION = edge
-TAG = $(VERSION)
-PREFIX ?= nginx-kubernetes-gateway
-
-GIT_COMMIT = $(shell git rev-parse HEAD)
+GIT_COMMIT = $(shell git rev-parse HEAD || echo "unknown")
 DATE = $(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
 
-TARGET ?= local
-
-KIND_KUBE_CONFIG_FOLDER = $${HOME}/.kube/kind
-OUT_DIR=$(shell pwd)/build/.out
+# variables that can be overridden by the user
+PREFIX ?= nginx-kubernetes-gateway## The name of the image. For example, nginx-kubernetes-gateway
```

Comment:
Can we add a space between the variable and the comment for all of these?

Reply:
Added spaces